content/mondoo-vmware-inventory.mql.yaml

```diff
@@ -50,52 +50,52 @@ packs:
           vsphere.datacenters { vms { name advancedSettings["guestInfo.detailed.data"] properties["guest"]["guestState"] } }
       - uid: mondoo-vmware-asset-inventory-esxi-kernel-modules
         title: VMware ESXi Kernel modules
-        filters: asset.platform == "vmware-esxi"
```

Comment:
This does not seem to be correct. Why do we change the asset type? The resources are defined here https://github.com/mondoohq/cnquery/blob/main/providers/vsphere/resources/vsphere.lr#L107-L113